### DISCUSSION TITLE
.NET nuget package update

### DISCUSSION THREAD

**Discussion by gorillapower** [scalar/scalar#2499] [Q&A]
As per this issue #2368 , the configuration is broken in the current nuget package (1.1.1) for .NET integration.

Is there any chance that the nuget could be bumped to the latest version?

**Comment by hanspagel** [comment]
Oh yes, I think @marclave needs to do this manually as we didn't set up any automation yet. 🙈

**Comment by marclave** [comment]
just released a new version!

@hanspagel we should totes automate this haha

https://www.nuget.org/packages/Scalar.AspNetCore/1.1.2

**Comment by xC0dex** [comment]
Finally, it's fully automated 🥳
